docs/conf.py

```diff
@@ -19,6 +19,7 @@
 # ones.
 extensions = [
     "sphinx.ext.todo",
+    "sphinx_rtd_dark_mode"
```

Comment:
This module stores a pref that persists in local storage - it would be better not to have this present.